layouts.py

```diff
@@ -240,15 +239,21 @@ def volatility(price, period_value, data_interval):
        a pandas series of volatility measures
     """
 
-    if data_interval not in ["1T", "30T", "1D"]:
+    if data_interval not in ["1T", "30T", "1D", "30D", "60D", "7D"]:
         raise ValueError
 
     if data_interval == "1T":
         trading_periods = 60 * 24
     elif data_interval == "30T":
         trading_periods = 2 * 24
-    else:
+    elif data_interval == "1D":
         trading_periods = 1
+    elif data_interval == "30D":
+        trading_periods = 30
+    elif data_interval == "60D":
+        trading_periods = 60
+    elif data_interval == "7D":
+        trading_periods = 60
```

Comment:
@Cyrusb01 I think this is an error, should be 7 if the first two are correct.

Reply:
It should be 7, but I actually don't use that, in that function 1D is the only thing ever used